src/module_TOUCHSCREEN/SerialOutput.java

```diff
@@ -0,0 +1,26 @@
+package module_TOUCHSCREEN;
```

Comment:
Is this package the most appropriate place for this? It seems more related to a PI Module rather than a TOUCHSCREEN Module. (This same comment holds true for all of the related files in this same package)